scipy/stats/_continuous_distns.py

```diff
@@ -5457,7 +5459,8 @@ class recipinvgauss_gen(rv_continuous):
 
     .. math::
 
-        f(x, \mu) = \frac{1}{\sqrt{2\pi x}} \frac{\exp(-(1-\mu x)^2}{2x\mu^2)}
+        f(x, \mu) = \frac{1}{\sqrt{2\pi x}}
```

Comment:
Seems like what was intended according to:
https://onlinelibrary.wiley.com/doi/10.1002/9781118445112.stat01207